### PR TITLE
Ww 159

### PR DESCRIPTION
**Pull request by cormorando**

### CODE REVIEW

File: extensions/wikia/ContributionAppreciation/styles/UserContributionAppreciation.scss [hunk 1]
```diff
@@ -10,21 +13,33 @@
...
-			fill: $color-buttons;
+			fill: $color-wikia-light-blue;
 			height: 48px;
 			object-fit: contain;
 			padding: 24px;
 			width: 48px;
 		}
 	}
 
+	.oasis-dark-theme & {
```
Comment (lukaszkonieczny) [inline extensions/wikia/ContributionAppreciation/styles/UserContributionAppreciation.scss]: we have variable `$is-dark-wiki` so you can set proper values to variables like `$banner-bg-color`

File: extensions/wikia/ContributionAppreciation/styles/UserContributionAppreciation.scss [hunk 2]
```diff
@@ -1,7 +1,10 @@
 @import 'skins/shared/color';
 
+$color-dark-gray: #333;
+$color-off-white: #f6f6f6;
```
Comment (rybmat) [inline extensions/wikia/ContributionAppreciation/styles/UserContributionAppreciation.scss]: these two colors are defined already: 
https://github.com/Wikia/style-guide/blob/dev/src/scss/lib/_settings.scss#L91
https://github.com/Wikia/style-guide/blob/dev/src/scss/lib/_settings.scss#L88

**Comment by sebastianmarzjan** [comment]
Given we use hardcoded colors in:
https://github.com/Wikia/app/blob/dev/extensions/wikia/BannerNotifications/css/BannerNotifications.scss
and already have a ticket to sunset this time-bound prototype:
https://wikia-inc.atlassian.net/browse/WW-123

LGTM

**Comment by sebastianmarzjan** [comment]
:+1: - nice simplification!

**Comment by dianafa** [comment]
LGTM please rebase from dev to get rid of those failing tests

**Pull request merged**